portfolio/src/main/webapp/js/custom-script.js

```diff
@@ -95,15 +94,81 @@ function loadMap() {
   });
 }
 
-function codeAddress(address) {
-  geocoder.geocode({'address': address}, function(results, status) {
+/**
+ * Places a marker on the current loaded map at the given location with an
+ * information window
+ * @param {String} address Postal address of the location you would like to
+ *     place a marker on the map for
+ */
+function placeMarker(nameAndLocation) {
+  geocoder.geocode({'address': nameAndLocation[1]}, function(results, status) {
+    if (status == 'OK') {
+      map.setCenter(results[0].geometry.location);
+      const marker = new google.maps.Marker({
+        map: map,
+        position: results[0].geometry.location,
+        animation: google.maps.Animation.DROP,
+      });
+
+      const content = '<div id="content">' +
+          '<div id="siteNotice">' +
+          '</div>' +
+          '<div id="bodyContent">' +
+          '<b>' + nameAndLocation[0] + '</b><br>' + nameAndLocation[1] +
+          '</div>' +
+          '</div>';
```

Comment:
It's a bummer that standard javascript doesn't have [templated strings](https://developers.google.com/web/updates/2015/01/ES6-Template-Strings) aka printf. This is kind of an eyesore to me but I assume this it's the cleanest way available.

Reply:
banilla javascript doesn't have this feature? i'll look into a better string formatting method.